libparsec/Cargo.toml

```diff
@@ -36,3 +36,8 @@ libparsec_serialization_format = { workspace = true }
 libparsec_testbed = { workspace = true, optional = true }
 
 dirs = { workspace = true }
+
+# CLI dependencies
```

Comment:
Should we have a separate bin crate `libparsec_cli` ?

Currently `libparsec` is used as a point of centralization of crate to import and is used notably for wasm32 bindings which won't support tokio.

Reply:
I can configure for `not(wasm32)` this is not a problem

Reply:
Yes with `not(wasm32)` that would solve that issue.

Now I think about the electron binding, with feature normalization that may cause problem in the future (especially with `tokio`)

Reply:
This can be also done under a feature